package.json

```diff
@@ -96,6 +96,7 @@
     "react": "^15.3.1",
     "react-addons-transition-group": "^15.3.0",
     "react-addons-update": "^15.3.1",
+    "react-copy-to-clipboard": "^5.0.0",
```

Comment:
Yep I can support that. @zanemountcastle's arguments with only the editors using it are accurate.

Maybe add a `/* TODO */` or an issue regarding where you hide the copy button that makes us add a banner at the top if the browser doesn't support it letting them know they're using a legacy browser and that it would be beneficial to them upgrading to the newest version for full capabilites.